pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ASTBlockStatement.java

```diff
@@ -4,9 +4,11 @@
 
 package net.sourceforge.pmd.lang.apex.ast;
 
-import com.google.summit.ast.Node;
 import net.sourceforge.pmd.annotation.InternalApi;
 
+import com.google.summit.ast.Node;
```

Comment:
Our style requirement is to have (non-static) imports in a single block (i.e. no whitespace) in ASCII sort order.

If PMD has a style, follow that, otherwise ours.

Reply:
PMD's import order is defined by the [Checkstyle](https://github.com/pmd/build-tools/blob/master/src/main/resources/net/sourceforge/pmd/pmd-checkstyle-config.xml#L89) plugin